tools/topology/topology2/cavs/platform/intel/deep-buffer.conf

```diff
@@ -0,0 +1,39 @@
+
+	Object.Pipeline {
+		deepbuffer-playback."1" {
+			index $DEEP_BUFFER_PIPELINE_ID
+
+			Object.Widget.copier.1 {
+				stream_name 'Deepbuffer Playback'
+			}
+
+			Object.Widget.gain.1 {
+				Object.Control.mixer.1 {
+					name 'Playback Volume 1'
```

Comment:
we need to start thinking of more meaningful control names. it's rather useless if 'Playback Volume 1' is exposed to userspace.

Reply:
sure , changed